source/renderer/app/utils/walletUtils.js

```diff
@@ -1 +1,13 @@
+// @flow
+import BigNumber from 'bignumber.js';
+
+// $FlowFixMe TODO: fix this in flowconfig
 export default import('@iohk-jormungandr/wallet-js').then((modules) => modules);
+
+const MINIMUM_BALANCE_FOR_REWARD: number = 10;
```

Comment:
@danielmain here we need to add a comment about the unit - we are talking about 10 ADA and not 10 lovelaces.
Please see these examples: https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/config/stakingConfig.js#L98-L99

Reply:
Also, please rename this to `MINIMUM_BALANCE_FOR_WITHDRAWING_REWARDS`

Reply:
No comment pls, we can add MINIMUM_**ADA**_BALANCE_FOR_WITHDRAWING_REWARDS. ok?

Reply:
Please add the comment. This is our standard for this type of declarations. Thanks!

Reply:
I will add the comment but pls 🙏 check this https://www.oreilly.com/library/view/clean-code-a/9780136083238/chapter04.html

Reply:
Thanks!

Reply:
Is this comment ok for you? // Minimal balance amount of ADA

Reply:
This comment means I have taken note of your suggestion and will consider it with the rest of the team when the time is right. For now, please keep the code consistent and add the comment 🙏